test/unit/services/impersonationRequests.test.ts

```
@@ -3,26 +3,27 @@
+import { REQUEST_DOES_NOT_EXIST, TASK_REQUEST_MESSAGES } from "../../../constants/requests";
 import userDataFixture from "../../fixtures/user/user";
-import cleanDb from "../../utils/cleanDb";
 import { impersonationRequestsBodyData } from "../../fixtures/impersonation-requests/impersonationRequests";
 import { CreateImpersonationRequestModelDto } from "../../../types/impersonationRequest";
 import { Timestamp } from "firebase-admin/firestore";
+
+const authService = require("../../../services/authService");
 const userQuery = require("../../../models/users");
 
 describe("Tests Impersonation Requests Service", () => {
   let mockRequestBody: CreateImpersonationRequestModelDto = impersonationRequestsBodyData[0];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'mockRequestBody' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)